bigquery/main.go

```diff
@@ -74,6 +74,7 @@ type bqRow struct {
 	Tags           []string
 	Env            []string
 	LogURL         string
+	Substitutions  [][]string
```

Comment:
Why is this a slice of slices instead of a slice of maps? ie: []map[string]string

Reply:
since BigQuery doesn't support map based structures for its schema, the map will be replaced with key value pairs instead, with each key value pair being represented as a string slice of size two

Reply:
Sounds good